benches/bench_rules.rs

```diff
@@ -84,8 +84,6 @@ fn list_parse(c: &mut Criterion) {
 fn get_blocker(rules: impl IntoIterator<Item=impl AsRef<str>>) -> Blocker {
     let (network_filters, _) = adblock::lists::parse_filters(rules, false, Default::default());
 
-    println!("Got {} network filters", network_filters.len());
```

Comment:
I'd prefer to comment this line as done in other benches. Or may be leave it as is If it doesn't break anything

Reply:
It actually breaks a bench with `--output-format bencher`